src/App.css

```diff
@@ -1,39 +1,30 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+table {
+  width: 50% !important;
+  font-size: 25px;
+  margin: 50px auto;
+  color: white;
```

Comment:
- [ ] All the boilerplate that got created by the CRA (texts, images, and styles) need to be deleted. Example: logos, favicon

Reply:
`Please update your readme file and add the comment that the users can run in order to test your app`
Please clarify what you mean, I don't get it.

Reply:
That was my mistake sorry, it wasn't meant to be there